CareKit/CareKitTests/TestAnyEventStoreExtensions.swift

```diff
@@ -118,10 +118,6 @@ class TestAnyEventStoreExtensions: XCTestCase {
             didUpdate.fulfill()
         }
 
-        #if compiler(>=5.8.0)
```

Comment:
I believe this will cause the CI not to build on Xcode 14.2 because fulfillment isn’t available. We can see what happens with the merge